test.py

```diff
@@ -2,43 +2,37 @@
 
 
 
-if True:
-    pass
+pass
 
 
 def function():
-    if False:
-        pass
-
-    if x == None:
+    if x is None:
         do_x()
 
     return None
-    do_x()
```

Comment:
Function function refactored with the following changes:

- Remove redundant conditional
- Remove unreachable code
- Use x is None rather than x == None